modules/nlu/src/views/lite/intentEditor/SlotPopover.tsx

```diff
@@ -8,7 +8,7 @@ import style from './style.scss'
 const MENU_WIDTH = 300
 
 export const TagSlotPopover = props => {
-  if (!props.show) {
+  if (!props.show || !window.getSelection().rangeCount) {
```

Comment:
Oh right, I had this before in the onSelectionChange method of the utterance editor. It was supposed to be moved here but seems like it got lost in the wild. Good catch!